src/include/Telnyx/UTL/Thread.h

```diff
@@ -15,6 +15,11 @@
 
 #include "Telnyx/Telnyx.h"
 
+// Forward declaration
+namespace Poco
```

Comment:
Follow the namespace convention, move the open bracket on the same line with namespace

Reply:
done, thanks!